benchadapt/python/benchadapt/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.1"
+__version__ = "23.2.8"
```

Comment:
If we're going to use calversioning (which I'm totally fine with) could we please use `YYYY.MM.DD` (or `YYYY.M0.D0` if we really hate the month and date 0 padding) so that it's clear(er) that this isn't a semvar?

Reply:
_technically_ it's possible we're on the 2023rd major release, of course, but I think seeing 2023. will trigger year more than 23. in someone's eyes

Reply:
We definitely can use the full year. I was a little hesitant about 0-padding because of https://peps.python.org/pep-0440/#integer-normalization . In some cases, the leading 0 will be stripped, which could be confusing. I also slightly think `2023.2.8` looks more like a date than `2023.20.80` but I will definitely not die on that hill!

Reply:
This might have been a typo, but in case not: the padding would go on the left, so `2023.02.08`, but yeah those zeros are definitely less important than `YYYY.`

Reply:
Yep, I looked at that site, and that's what convinced me that 0-padding is scary! Because it claims that `youtube_dl` uses 0M.0D (ex https://github.com/ytdl-org/youtube-dl/blob/2021.06.06/youtube_dl/version.py) but PyPI remembers those version numbers without 0-padding (ex. https://pypi.org/project/youtube_dl/#history)